src/estimators/any_sketch.py

```diff
@@ -317,8 +353,18 @@ def add(self, x):
     indexes = self.get_indexes(x)
     # Move this to the loop when we support more than one value function.
     value_function = self.config.value_functions[0]
+
+    if value_function == UniqueKeyFunction():
+      assert self.config.num_hashes == 1, (
+          'Same Key Aggregator only supports one hash function.')
+      assert isinstance(x, int) and (x >= 0) and (x < 2 ** 31), (
+          'Now for conveninence, Same Key Aggregator only supports'
+          'adding integer np.int32 ids.')
+      value_to_add = x
+    else:
+      value_to_add = 1
     for index in indexes:
-      self.sketch[index] = value_function(self.sketch[index], 1)
+      self.sketch[index] = value_function(self.sketch[index], value_to_add)
```

Comment:
I think it would be better to add a method on value function that takes the input value, "x" and returns the value to add. The existing ones can just return 1. Maybe call that method "get_value_from_fingerprint"?

Reply:
good suggestion, added!